syntax = "proto3";

package user;

option go_package = "/proto;user";

service UserService {
    rpc ListUsers (Empty) returns (UserList);
}

message Empty {}

message User {
    int32 id = 1;
    string username = 2;
    string email = 3;
    string fullname = 4;
}

message UserList {
    repeated User users = 1;
}